src/types/index.ts

```diff
@@ -6,14 +6,13 @@ import { Config, ConfigurationRoutes, ConfigurationGenericProperty } from './cli
 */
 
 // Type based of the manifest.json file in the templates folder
-export interface Manifest{
+export interface Manifest {
```

Comment:
I noticed there is a lot of these changes that are not specifically mentioned nor implied by your ticket. As best practice changed only what is **needed** for your assigned tasks. Its easier for reviews and also great for separation of concerns when working on projects